src/sql/plot.py

```diff
@@ -556,7 +556,7 @@ def _histogram(table, column, bins, with_=None, conn=None, facet=None):
 
         query = template.render(table=table, column=column, filter_query=filter_query)
 
-    data = conn.execute(query, with_).fetchall()
+    data = conn.execute(query, with_, read="mysql").fetchall()
```

Comment:
do we need to hardcode mysql?

Reply:
Removed the `read` parameter, we don't need that anymore